pytripgui/main.py

```diff
@@ -1,19 +1,21 @@
 import sys
 import argparse
-import logging
 
 from PyQt5.QtWidgets import QApplication
-
-from pytripgui.view.main_view import MainView
+from pytripgui.main_window_qt_vc.main_window_view import MainWindowQtView
+from pytripgui.main_window_qt_vc.main_window_cont import MainWindowController
 from pytripgui.model.main_model import MainModel
-from pytripgui.controller.main_cont import MainController
 
+
+import logging
 logger = logging.getLogger(__name__)
 
 
 class AppWindow():
     def __init__(self):
-
+        from pytripgui.view.main_view import MainView
```

Comment:
Why are you using imports inside function ?

Reply:
deleted this obsolete code

Reply:
Importing something inside the functions and not in the top part of the file is unusal in python and require some reason behind. What was the reason here ?